### ISSUE TITLE
Can't compile without GCP Storage

### ISSUE THREAD

**Issue by mbklein** [breakroom/sitemapper#34]
`google_api_storage` is listed as an optional dependency, but the module won't compile without it:

```
==> sitemapper
Compiling 11 files (.ex)
error: GoogleApi.Storage.V1.Model.Object.__struct__/1 is undefined, cannot expand struct GoogleApi.Storage.V1.Model.Object. Make sure the struct name is correct. If the struct name exists and is correct but it still cannot be found, you likely have cyclic module usage in your code
  lib/sitemapper/store/gcp_storage_store.ex:28: Sitemapper.GCPStorageStore.write/3


== Compilation error in file lib/sitemapper/store/gcp_storage_store.ex ==
** (CompileError) lib/sitemapper/store/gcp_storage_store.ex: cannot compile module Sitemapper.GCPStorageStore (errors have been logged)
```

Easy enough to work around by adding the dependency, but it'd be nice to keep it optional to avoid bloat.

**Comment by tomtaylor** [comment]
Ah yes, right. Maybe I’ll move this into a separate package.

**Comment by tomtaylor** [comment]
@mbklein does #35 fix this for you?

**Comment by mbklein** [comment]
Yep! Works great. Thanks for addressing it so quickly.

**Comment by tomtaylor** [comment]
No worries! Closed by #35.
